app/views/organisations/merge.html.erb

```diff
@@ -0,0 +1,46 @@
+<% title = "Tell us if your organisation is merging" %>
+<% content_for :title, title %>
+
+<%= govuk_back_link href: organisation_path %>
```

Comment:
i see your screenshot is cleverly cropped. this needs to go into `before_content` to fix the spacing
```
content_for :before_content do
```

Reply:
Nice! Thanks!